test/index.js

```diff
@@ -4,75 +4,112 @@ var test = require('tape');
 
 var getSideChannel = require('../');
 
-test('export', function (t) {
-	t.equal(typeof getSideChannel, 'function', 'is a function');
-	t.equal(getSideChannel.length, 0, 'takes no arguments');
+var protos = [null, 'WeakMap', 'Map'];
```

Comment:
to ensure there's no breaking change, it would be ideal to make no changes to existing tests (only, perhaps, adding tests). Can you revert the changes to this file for now?

Reply:
Sure

Reply:
I added some tests and just made the change in a separate file, when satisfied there is no breaking change you can just replace index.js with local.js

Reply:
Sounds good.